polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,264 @@
+    - Polkadot (DOT): 10
+    - Kusama (KSM): 12
+- Base unit: "Planck"
+- Balance type: `u128`
+
+### What is DOT?
+
+DOT is the native token of the Polkadot network, much like BTC is for Bitcoin and Ether is for the Ethereum blockchain.
+
+### Redomination
+
+Polkadot conducted a community poll, which ended on **27 July 2020** at block **888,888** to decide whether to redenominate the DOT token. 
+
+The stakeholders chose to redenominate the token, changing the value of 1 DOT from 1e12 plancks to 1e10 plancks. 
+
+Importantly, this did not affect the total number of base units (plancks) in the network, only how a single DOT is represented.
+
+The redenomination became effective 72 hours after transfers were enabled, occurring at block **1,248,328** on 21 August 2020 around 16:50 UTC.
+
+## The Planck Unit
+
+The smallest unit of account balance on Substrate-based blockchains (such as Polkadot and Kusama) is called **Planck**, named after the **Planck Length**, the smallest measurable distance in the physical universe. 
+
+Similar to how BTC's smallest unit is the **Satoshi** and ETH's is the **Wei**, Polkadot’s native token DOT is equal to 1e10 Planck, while Kusama’s native token KSM equals 1e12 Planck.
+
+## What are the uses of DOT?
+
+DOT serves three primary functions within the Polkadot network:
+
+1. **Governance** - it is used to participate in the governance of the network
+2. **Staking** - DOT is staked to support the network’s operation and security
+3. **Buying Coretime** - used to purchase coretime in-bulk or on-demand and access the  chain to benefit from Polkadot's security and interoperability
+
+Additionally, DOT can serve as a transferable token. For example, DOT held in the Treasury can be allocated to teams developing projects that benefit the Polkadot ecosystem.
+
+## JAM and the Road Ahead
+
+The Join-Accumulate Machine (JAM) represents a transformative redesign of Polkadot's core architecture, envisioned as the successor to the current relay chain. Unlike traditional blockchain architectures, JAM introduces a unique computational model that processes work through two primary functions: Join (which handles the integration of data) and Accumulate (which folds computations into the chain's state). 
+
+Operating as a domain-specific rollup chain, JAM removes many of the opinions and constraints of the current relay chain while maintaining its core security properties. It's designed to be more generic and flexible, allowing for permissionless code execution through "services" that can be deployed without governance approval. What makes JAM particularly innovative is its transactionless nature and its efficient pipeline processing model, which places the prior state root in block headers instead of the posterior state root, enabling more effective utilization of block time for computations. This architectural evolution promises to enhance Polkadot's scalability and flexibility while maintaining its robust security guarantees.
```

Comment:
I am not sure this is entirely accurate, as can do more than that, I think we can just start the sentence straight away.
```suggestion
JAM removes many of the opinions and constraints of the current relay chain while maintaining its core security properties. It's designed to be more generic and flexible, allowing for permissionless code execution through "services" that can be deployed without governance approval. What makes JAM particularly innovative is its transactionless nature and its efficient pipeline processing model, which places the prior state root in block headers instead of the posterior state root, enabling more effective utilization of block time for computations. This architectural evolution promises to enhance Polkadot's scalability and flexibility while maintaining its robust security guarantees.
```